DEVELOPING.md

```diff
@@ -0,0 +1,45 @@
+## Pre-requisites
+
+1.  We are using Node 8. If you need to work with multiple versions of Node, you
+    might consider using [nvm](https://github.com/creationix/nvm).
+1.  This repository uses [yarn](https://yarnpkg.com/) to manage node dependencies. Please install yarn globally using `npm install --global yarn`.
+
```

Comment:
Contributing
1. Familiarize yourself with the codebase and the development doc.
2. Create a new issue before starting your project so that we can keep track of what you are trying to add/fix. That way, we can also offer suggestions or let you know if there is already an effort in progress.
3. Fork this repository.
4. Create a topic branch in your fork based on the correct branch (usually the develop branch, see Branches section below). Note, this step is recommended but technically not required if contributing using a fork.
4. Edit the code in your fork.
5. Sign CLA (see CLA below)
6. Send us a pull request when you are done. We'll review your code, suggest any needed changes, and merge it in.